Modules/Core/Common/include/itkBooleanStdVector.h

```diff
@@ -34,7 +35,8 @@ namespace itk
  * \note BooleanStdVectorType is only intended to be used for the storage of Boolean values (true and false), not for
  * arbitrary integer values.
  */
-using BooleanStdVectorType = std::vector<uint8_t>;
+
```

Comment:
The documentation (above here) is specifically about `BooleanStdVectorType`, so I'm not sure if it's handy to add an extra white line between the documentation and the definition of `BooleanStdVectorType`.

Reply:
Will Doxygen then still understand that they belong together?

Reply:
Will fix.

Reply:
Almost running out of nitpicks  ;-)